social-network/src/components/ArticleCard.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+
+function ArticleCard(props) {
+  const monList = ['Jan', 'Feb', 'Mar', 'Apr', 'May', 'Jun','Jul', 'Aug', 'Sep', 'Oct', 'Nov', 'Dec']
+  const newArtDate = new Date(props.article.postedDate)
+  const newDate = newArtDate.getDate();
+  const newMonth = newArtDate.getMonth();
+  let artMon = '';
```

Comment:
Another method that's faster and a little more readable is `const artMon = monList[newMonth]`, since the index from `getMonth` aligns with the months in the list you defined.